Chrysalis-Papilio/Soen490-Papilio-Website
# PAPILIO-99-bug

## General 
React-refresh a dependencies of multiple of the packages that we are using is being loaded in multiple versions. Npm does not like that. We have to fix this discrepancies between the different packages and use a single version so that we will make our lord npm happy.
At the same time, it would be good to remove the unnecessary white space being introduced by the graph on the dashboard home page.

_Issue number_: Papilio-99

## Task description
- Force npm package version
- Remove unnecessary whitespace
- Cleanup asynchronous tests

## Manual Test
Make sure that you have the appropriate variables in the backend .env file. (Contact me if you don't)

1. start the backend
`npm run build && npm start`
2. start the website (it should be running to prove the npm conflict solution)
`npm start`
3. login to the dashboard
4. verify the whitespaces in the different pages of the dashboard

## Testing
To run all the unit tests, run the following code: `npm test`
To make sure that the tests have reach the appropriate level of coverage, run the following code: `npm run coverage`
